### DISCUSSION TITLE
Sonic 3 and knuckles act transitions

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#74] [Q&A]
I am trying to add Sonic 3 and knuckles act transitions but when I did it added the scene and overlapped the first zone

**Comment by DimensionWarped** [comment]
This is a repeat of https://github.com/Techokami/SonicWorldsNext/discussions/55

We don't have solid answers on it yet. Please consult the Discussions list to make sure whether or not what you are looking for already exists before making new threads in the future.
